pkg/runtest/run.go

```diff
@@ -178,7 +178,7 @@ func (ctx *Context) generatePrograms() error {
 	}
 	for _, file := range files {
 		if err := ctx.generateFile(sandboxes, cover, file); err != nil {
-			return err
+			continue
```

Comment:
This looks fishy.
If we ignore the error, we shouldn't assign and check error in the first place.
This is probably the only call to generateFile, which means it shouldn't return errors at all. But there sure bad things that can happen and must not be ignored, e.g. failures to read files.

Reply:
There are other cases in which generateFile() returns an error: namely, if it cannot create a syztest/C test.
Maybe we should return `nil` in the case of a `parseProg()` failure (now that we create `runRequest` instances for it), and log other errors:
```
  ctx.log("generateFile(%v) returned error: %v\n", file, err)
  continue
```
?

I still think that bailing out of that loop is not the best idea, as the error value of `generatePrograms()` is ignored, and we end up just partially executing the test corpus, potentially still treating it as a success.